standard_icone_1246_1639063513.json

```diff
@@ -0,0 +1,310 @@
+{
```

Comment:
This file and the one above (standard_icone_1245_xxx.json) seem to also exist under wzdx/sample_files/standard/icone. Can we remove from root?

Reply:
removed